gnome-initial-setup/pages/account/um-photo-dialog.c

```diff
@@ -195,36 +192,48 @@ create_face_widget (gpointer item,
 }
 
 static GSList *
-get_facesdirs (void)
+get_settings_facesdirs (void)
 {
+        g_autoptr(GSettings) settings = NULL;
+        g_autoptr(GSettingsSchema) schema = NULL;
+
         GSList *facesdirs = NULL;
-        const gchar * const * data_dirs;
+        char **settings_dirs;
         int i;
-        const char *facesdir_env;
-        GisDriver *driver;
 
-        facesdir_env = g_getenv ("GIS_ACCOUNT_PAGE_FACESDIR");
-        if (facesdir_env != NULL && g_strcmp0 (facesdir_env, "") != 0) {
-                facesdirs = g_slist_prepend (facesdirs,
-                                             g_strdup (facesdir_env));
+        GSettingsSchemaSource *source = g_settings_schema_source_get_default ();
+
+        if (!source)
+                return facesdirs;
+
+        schema = g_settings_schema_source_lookup (source,
+                                                  "org.gnome.ControlCenter",
+                                                  FALSE);
+        if (!schema)
```

Comment:
I think this part should change now, I'm pretty sure you can assume that the `org.gnome.ControlCenter` schema is present, but you should check for the key and return `facesdirs` in that case so that you don't abort if the FBE is used with a mismatched version of Settings.

Reply:
Done